plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManagerTest.kt

```diff
@@ -201,11 +202,20 @@ class LocationPuckManagerTest {
 
   @Test
   fun testShow() {
+    locationPuckManager.isHidden = true
     locationPuckManager.show()
     verify { locationLayerRenderer.show() }
     assertFalse(locationPuckManager.isHidden)
   }
 
+  @Test
```

Comment:
we're missing the test to cover scenario for implicit `locationPuckManager.show(forceUpdate = true)`, aren't we?

Reply:
Added a test case